rfc/0053-sdk3-crud.md

```diff
@@ -1188,9 +1188,18 @@ public interface ILookupInResult : IResult {
 }
 ```
 
-- Exists(..) will return true if the item exists for a given index, otherwise false.
-
-- ContentAs will throw a NotFoundException if the item does not exist; if the index is invalid a InvalidIndexException will be thrown.
+- Exists
+  1. Throws InvalidIndexException if the index is < 0 or >= the number of lookups.
+  2. Returns true if the status code at the index is `SUCCESS` or `SUCCESS_DELETED_DOCUMENT`.
```

Comment:
I don't think that SUCCESS_DELETED_DOCUMENT can be returned by an individual subdocument operation can it?  Wouldn't that appear at the top level operation level?

Reply:
You are correct. Updated.